packages/eslint-plugin/src/rules/emotion-to-compiled/__tests__/unit/rule.test.tsx

```diff
@@ -0,0 +1,75 @@
+import type { RuleTester } from 'eslint';
```

Comment:
Can you remove `unit/` in the file path. That convention is not yet used in this repository.

Reply:
yep